lib/framework/data_context.dart

```diff
@@ -311,7 +312,8 @@ class NativeInvokable with Invokable {
       ActionType.stopTimer.name: stopTimer,
       ActionType.openCamera.name: showCamera,
       ActionType.navigateBack.name: navigateBack,
-      'debug': (value) => log('Debug: $value')
+      'debug': (value) => log('Debug: $value'),
```

Comment:
can you check if `log('...')` works in release mode? As in does the browser in Web release mode even print it out? If not use print() or something else.

Reply:
its working in release mode